frontend/packages/volto-light-theme/src/components/Theming/Theming.tsx

```diff
@@ -48,7 +48,7 @@ const Theming = ({ content }: { content: Content }) => {
   const addingFromInherited =
     location.pathname.endsWith('/add') &&
     flattenToAppURL(
-      content?.['@components']?.inherit?.['voltolighttheme.theme'].from['@id'],
+      content?.['@components']?.inherit?.['voltolighttheme.theme']?.from['@id'],
```

Comment:
Did you test it? I think you also need:

```suggestion
      content?.['@components']?.inherit?.['voltolighttheme.theme']?.from?.['@id'],
```

and I'm also not sure what flattenToAppURL does if it gets passed `undefined`